### PR TITLE
Alicia/blog

### PR DESCRIPTION
**Pull request by AliciadW**
First iteration of blog list and blog pages.

@longhotsummer please review. I'll tweak and update styling as needed.

### CODE REVIEW

File: _layouts/about.html [hunk 1]
```diff
@@ -79,5 +79,6 @@ <h2>Funders</h2>
         </div>
       </div>
     </div>
+  </section>
```
Comment (longhotsummer) [inline _layouts/about.html]: :)

File: _layouts/blog-list.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+layout: slim
+---
+
+<section class="mb-5 py-3">
+  <div class="container">
```
Comment (longhotsummer) [inline _layouts/blog-list.html]: These container, row and col `div`s can be dropped completely, the slim layout forces the narrow column width for the whole page.

File: _layouts/blog-list.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="mb-5 py-3">
+  <div class="row">
+
+    {% for post in site.posts %}
+    <div class="card border-0 col-md-6 col-lg-4">
```
Comment (longhotsummer) [inline _layouts/blog-list.html]: On mobile, if you're half way through the list the space between the image and the into of two separate posts is the same as the space between a post's heading and it's image. So it means it's not visually clear if the title (or image) applies to the image (or title) above or below.

Basically: some extra `mb-3` on the card would help on mobile (and shouldn't hurt on desktop).

File: _layouts/blog-post.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+layout: simple
+---
+
+<section class="py-5">
+  <div class="container text-center">
+    <h2>{{ page.title }}</h2>
+    <p>{{ page.printDate }}, by <a href="#">{{ page.author }}</a></p>
```
Comment (longhotsummer) [inline _layouts/blog-post.html]: I don't think we'll have author links, so the author name doesn't need to be a link.

File: _layouts/blog-post.html [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    <img class="img-fluid my-5" src="https://bootstrapshuffle.com/placeholder/pictures/bg_16-9.svg" alt="">
```
Comment (longhotsummer) [inline _layouts/blog-post.html]: I think the image looks better the same width as the text column, so perhaps move this image element into the text column below.

I suspect `text-left` below is unnecessary.

**Comment by longhotsummer** [comment]
If you'd like a real post to test with, I'd like to pull over this post from the OpenUp blog. https://github.com/OpenUpSA/openupsa.github.io/blob/master/_articles/govt-law-cloud-computing.md

File: _layouts/blog-list.html (hunk 2 above)
Comment (AliciadW) [reply _layouts/blog-list.html]: Thanks, I've removed this.

File: _layouts/blog-list.html (hunk 3 above)
Comment (AliciadW) [reply _layouts/blog-list.html]: Done

File: _layouts/blog-post.html (hunk 4 above)
Comment (AliciadW) [reply _layouts/blog-post.html]: I've removed this.

File: _layouts/blog-post.html (hunk 5 above)
Comment (AliciadW) [reply _layouts/blog-post.html]: Done.

**Comment by AliciadW** [comment]
@longhotsummer I've updated styling and added the RSS feed

**Comment by longhotsummer** [comment]
This is looking great. Final touches:

* please replace `printDate` with something like `{{ post.date | date: "%d %b %Y" }}` (but so that it produces "15 August 2015") so that we don't need to provide the post's date twice.
* please remove the placeholder blog posts.

Then I think it's good to merge.

**Pull request merged**